content/docs/faq-state.md

```diff
@@ -18,6 +18,14 @@ Here are some good resources for further reading on when to use `props` vs `stat
 * [Props vs State](https://github.com/uberVU/react-guide/blob/master/props-vs-state.md)
 * [ReactJS: Props vs. State](http://lucybain.com/blog/2016/react-state-vs-pros/)
 
+### Is synching state and props a bad idea?
+
```

Comment:
I think it's not very clear from this description what "syncing" means.

I think we should show a three-line code example first and *then* explain why it's problematic.